client.sh

```diff
@@ -1,47 +1,21 @@
 #!/bin/bash
-#### This is a simple script that creates LTSP clients 
-#### These clients are diskless and only boot from network
-#### So make sure LTSP server is running before executing script
+#################################################
+#  Simple script that creates LTSP clients      #
+#  These clients can only boot from network     #
+#  LTSP server is running before running script	#
```

Comment:
just remove white space and correct alignment so it will not look weird.

Reply:
I dont know how but, it only looks weird here.
if you check here - https://github.com/docker-scripts/dev--LTSP/blob/enhance/client.sh
or my editor you wont see the white space.